test/prism/magic_comment_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative "test_helper"
+
+module Prism
+  class MagicCommentTest < TestCase
+    examples = [
+      "# encoding: ascii",
+      "# coding: ascii",
+      "# eNcOdInG: ascii",
+      "# CoDiNg: ascii",
+      "# \s\t\v encoding \s\t\v : \s\t\v ascii \s\t\v",
+      "# -*- encoding: ascii -*-",
+      "# -*- coding: ascii -*-",
+      "# -*- eNcOdInG: ascii -*-",
+      "# -*- CoDiNg: ascii -*-",
+      "# -*- \s\t\v encoding \s\t\v : \s\t\v ascii \s\t\v -*-",
+      "# -*- foo: bar; encoding: ascii -*-",
+      "# coding \t \r  \v   :     \t \v    \r   ascii-8bit\n",
+      "# vim: filetype=ruby, fileencoding=big5, tabsize=3, shiftwidth=3\n"
```

Comment:
Why only these lines  have a newline?

```suggestion
      "# coding \t \r  \v   :     \t \v    \r   ascii-8bit",
      "# vim: filetype=ruby, fileencoding=big5, tabsize=3, shiftwidth=3",
```

Reply:
@nobu removed in https://github.com/ruby/prism/commit/4a7be1fc507076fd67d4a003946736382953c079